threedi_scenario_downloader/downloader.py

```diff
@@ -177,7 +183,7 @@ def get_raster(scenario_uuid, raster_code):
 
 
 def create_raster_task(
-    raster, scenario_instance, resolution=None, projection=None, bbox=None, time=None
+    raster, scenario_instance, projection=None, resolution=None, bbox=None, time=None
```

Comment:
Wordt dit altijd met keyword arguments gebruikt? Of worden resolution/projection ook wel als "positionele parameters" meegegeven? *In dat laatste geval* moet je wel even wat in de `CHANGES.rst` zetten (aannemende dat die bestaat) omdat het dan een verandering is waar gebruikers van moeten weten.